bfh/fields.py

```diff
@@ -63,7 +64,10 @@ def __get__(self, instance, cls=None):
         return instance.__dict__.get(self.field_name)
 
     def __set__(self, instance, value):
-        instance.__dict__[self.field_name] = value
+        if value is None:
```

Comment:
shouldn't the default be on __get__, not __set__? It seems like odd, stateful behavior to require that None has been passed to __set__ for the default value to appear. In other words, i think you need a test like this (adapting what you have below):

        s = DefaultsSchema().serialize()
        self.assertEqual(s.get("defaulted"), "testing")